frontend/src/index.css

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer utilities {
+  .dark {
```

Comment:
What is the use case of this ? And why is this added ?

Reply:
sir should I keep dark or should I remove it

Reply:
I am asking the use case, I am trying to understand why have you added that, once you say, I can tell you if it is correct or if there is any other better way

Reply:
Sir I added .dark at global level it will automatically gives bg as black and text as white. This will help in adding dark mode at globel level.

Reply:
Its not convincing @SankalpHaritash21, you need to either convince why using this is best if you feel it is, or else we will remove it as it is used in only place if you can make it at global level, then you have to use it globally, i see it being used only at one place, so doesn't server the purpose of being global afterall

Reply:
Ok sir I will remove it and add it simply to  classes